src/bundle/Resources/public/scss/_content-type-edit.scss

```diff
@@ -60,6 +60,11 @@
                 }
             }
 
+            &--active-field-definitions-group {
+                border: calculateRem(1px) solid $ibexa-color-info;
+                border-radius: $ibexa-border-radius;
```

Comment:
Question: Shouldn't `border-radius` be defined directly on element? That border does not add to element width?